docs/ref/models/options.txt

```diff
@@ -101,6 +101,37 @@ Django quotes column and table names behind the scenes.
     setting, if set. If the backend doesn't support tablespaces, this option is
     ignored.
 
+``db_table_comment``
+--------------------
+
+.. versionadded:: 4.0
+.. attribute:: Options.db_table_comment
+
+A string describing the Model which will be saved as a SQL
+COMMENT on the SQL TABLE
+
+db_table_comments are displayed in the database as SQL
+COMMENTs, which makes them useful for documenting database
+tables for database administrators, developers, analysts, and others
+with direct database access who may not be looking at your Django
+code.
+For example::
+
+    db_table_comment = "Each entry in this table is gathered when\
+                           the user registers. This comment is managed by\
+                           Django, and will be updated on every migration."
+
+db_table_comment has a character limit that differs for each database
+backend. PostgreSQL has a 1GB limit. MySQL has a 2048-character limit.
+
+.. warning::
+    Changes to database table comments made by database
+    users outside of Django will be overwritten by Django migrations
+    if Django finds a value for db_table_comment on a model.
+    Communicating with your DBAs and issuing reminder notices in
+    the comments (as in the example above) may help avoid conflicts.
```

Comment:
I don't think this warning is necessary - this is how Django works for everything else, I don't know if we need to call it out for comments specifically.